packages/e2e-tests/specs/experiments/multi-entity-editing.test.js

```diff
@@ -152,10 +152,7 @@ describe( 'Multi-entity editor states', () => {
 	const templatePartName = 'Test Template Part Name Edit';
 	const nestedTPName = 'Test Nested Template Part Name Edit';
 
-	useExperimentalFeatures( [
-		'#gutenberg-full-site-editing',
-		'#gutenberg-full-site-editing-demo',
-	] );
+	useExperimentalFeatures( [ '#gutenberg-full-site-editing' ] );
```

Comment:
I suspect that the changes here are probably going to impact the e2e tests, we should rework those to rely on 2021-blocks instead. (active the theme at the beginning and disable it in the end)

Reply:
> we should rework those to rely on 2021-blocks instead. (active the theme at the beginning and disable it in the end)

That makes sense to me.  I would expect the tests would still work after we ensure that theme is enabled.

Reply:
If 2021-blocks is loaded automatically in wp-env config, wouldn't:

> active the theme at the beginning and disable it in the end

be unnecessary for the test?  Or is it common that other themes are activated/deactivated in other e2es?

Reply:
Regardless of what theme we enable by default, we might have tests that rely on regular themes to be enabled (like today), so we can try enabling it by default and see.